roxie/ccd/ccdprotocol.cpp

```diff
@@ -57,7 +57,7 @@ class CHpccProtocolPlugin : implements IHpccProtocolPlugin, public CInterface
         trapTooManyActiveQueries = ctx.ctxGetPropBool("@trapTooManyActiveQueries", true);
         numRequestArrayThreads = ctx.ctxGetPropInt("@requestArrayThreads", 5);
         maxHttpConnectionRequests = ctx.ctxGetPropInt("@maxHttpConnectionRequests", 10);
-        maxHttpKeepAliveWait = ctx.ctxGetPropInt("@maxHttpKeepAliveWait", 5000);
+        maxHttpKeepAliveWait = ctx.ctxGetPropInt("@maxHttpKeepAliveWait", 5000); // MORE - is this supposed to be 5000 seconds? Because that's what it seems to be interpreted as
```

Comment:
Right, so this is the amount of time we will wait to see if another request arrives on the same connection.  It can certainly be made shorter than 83 1/3 minutes.

Reply:
What would be a reasonable value? 5 seconds?

Reply:
Yes that would be good.  Probably the original intent of the 5000.